src/Microsoft.ML.Transforms/OneHotHashEncoding.cs

```diff
@@ -18,8 +18,12 @@
 namespace Microsoft.ML.Transforms
 {
     /// <summary>
-    /// Produces a column of indicator vectors. The mapping between a value and a corresponding index is done through hashing.
+    /// <see cref="ITransformer"/> resulting from fitting a <see cref="OneHotHashEncodingEstimator"/>.
     /// </summary>
+    /// <remarks>
+    /// CategoricalHashOneHotVectorizer converts a categorical value into an indicator array by hashing the value and using the hash as an index in the bag.
```

Comment:
What is CategoricalHashOneHotVectorizer? if this class has been renamed, you could just say:
"Convert a categorical value into an indicator array by ..."

Reply:
Old name of this transformer. Missed that.